.github/workflows/docker_image_scan.yml

```diff
@@ -0,0 +1,54 @@
+name: Scan docker image
+on:
+  schedule:
+    - cron: '0 7 * * *'
+
+jobs:
+  scan-docker-image:
+    runs-on: ubuntu-latest
+    name: Scan docker image
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Build docker image
+      run: |
+        docker build \
+          --tag cccd:scan \
+          --file docker/Dockerfile .
+    - name: Scan docker image using snyk
+      id: image-scan
+      continue-on-error: true
+      uses: snyk/actions/docker@master
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      with:
+        image: cccd:scan
+        args: --file=docker/Dockerfile
+    - name: Upload result to GitHub Code Scanning
+      uses: github/codeql-action/upload-sarif@v1
+      with:
+        sarif_file: snyk.sarif
+    - name: Output outcome
+      run: |
+        echo ${{ steps.image-scan.outcome }}
+    - name: Slack notify failure
+      if: ${{ steps.image-scan.outcome == 'failure' }}
+      uses: rtCamp/action-slack-notify@v2
+      env:
+        SLACK_USERNAME: Snyk docker image scan
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+        SLACK_CHANNEL: laa-cccd-alerts
+        SLACK_ICON_EMOJI: ':snyk:'
+        SLACK_COLOR: '#fc0303'
```

Comment:
(nit) If there is a 'success' colour (below) is there also a named colour for failure?

Reply:
I think it was purple or I could not find one and purple was the default. I forget now. We can tweak on a separate branch i'd say

Reply:
>        SLACK_COLOR: ${{ job.status }} # or a specific color like 'good' or '#ff00ff'
https://github.com/rtCamp/action-slack-notify/blob/28e8b353eabda5998a2e1203aed33c5999944779/README.md